builtin/plugins/coin/coin.go

```diff
@@ -76,7 +79,40 @@ func (c *Coin) Meta() (plugin.Meta, error) {
 func (c *Coin) Init(ctx contract.Context, req *InitRequest) error {
 	div := loom.NewBigUIntFromInt(10)
 	div.Exp(div, loom.NewBigUIntFromInt(18), nil)
-
+	//Checks if Coin Policy Feature is enabled before loading Coin Monetary Supply Policy
+	if ctx.FeatureEnabled(loomchain.CoinPolicyFeature, false) {
```

Comment:
The `Init` only runs once when the contract is deployed, there's not much point in checking for this feature flag here. Better to just skip these validation checks & skip calling `ctx.Set(policyKey, policy)` if `req.Policy == nil`.

Reply:
Done